src/main/kotlin/com/neva/gradle/fork/config/Config.kt

```diff
@@ -135,17 +139,21 @@ abstract class Config(private val fork: ForkExtension, val name: String) {
       return
     }
 
-    val fileProps = Properties()
-    fileProps.load(FileInputStream(file))
-    fileProps.forEach { p, v -> prompts[p.toString()]?.value = v.toString() }
+    FileInputStream(file).use { input ->
```

Comment:
closeable not used to that it was not possible to delete `build/fork/config/properties.properties` etc